editor.planx.uk/src/pages/Team/index.tsx

```diff
@@ -266,28 +244,10 @@ const Team: React.FC = () => {
             )}
           </>
         )}
-        {flows && !flows.length && <GetStarted flows={flows} />}
+        {flows && !flows.length && <GetStarted />}
       </Container>
     </Box>
   );
 };
 
 export default Team;
-
-const getUniqueFlow = (
```

Comment:
This functionality is now actually handled at runtime when trying to insert a flow as there's a unique constraint in place on the db column.